eng/WpfArcadeSdk/tools/SdkReferences.targets

```
@@ -210,4 +125,33 @@
       </Reference>
     </ItemGroup>
   </Target>
+
+  <ItemGroup>
+    <!-- We always want to use the local WindowsBase, not the one coming from the .NET runtime. -->
+    <DefaultReferenceExclusion Include="WindowsBase"/>
+  </ItemGroup>
+
+  <!--
+    Allows excluding specific .NET runtime libraries from being implicitly referenced.
+    For example, to exclude WindowsBase from being implicitly referenced, add the following to your project file:
+
+      <DefaultReferenceExclusion>
+        <MicrosoftPrivateWinFormsReference Include="WindowsBase" />
+      </DefaultReferenceExclusion>
```

This does not look like valid MSBuild syntax. Did you mean something like this ?
```suggestion
      <ItemGroup>
        <DefaultReferenceExclusion Include="WindowsBase" />
      </ItemGroup>
```

---

Copy past whoops.